cfgov/unprocessed/apps/tccp/css/main.less

```diff
@@ -23,6 +23,14 @@
   }
 }
 
+// Some cards list a lot of available locations and disrupt the table layout.
+// Cap the width of the availability cell on non-mobile screens to prevent this.
+@media only screen and (min-width: @bp-sm-min) {
+  td[data-label='Availability'] {
+    max-width: unit((400px / @btn-font-size), em);
```

Comment:
Any reason not to use `@base-font-size-px` here?

Reply:
Good catch! Auto-correct fail.